src/App.tsx

```diff
@@ -92,25 +100,39 @@ const router = createBrowserRouter([
       {
         path: paths.manufacturers,
         Component: ManufacturerLayout,
+        loader: manufacturerLayoutLoader(queryClient),
+        ErrorBoundary: ManufacturerLayoutErrorComponent,
```

Comment:
Not sure how react router handles this, but couldn't this be an `errorElement: ManufacturerErrorComponent` (or even just the error page directly) so that it uses existing layout when rendering it avoiding the need to also specifying the BaseLayoutHeader inside the error component as well.

Reply:
It doesn't work like that. The layout is only applied to the child routes.

Reply:
Ah yeah I see it would only have been the case if BaseLayout was at the root of the rest of the routes instead of inside of them.